app/src/main/res/layout/activity_profile.xml

```diff
@@ -0,0 +1,169 @@
+        android:layout_width="349dp"
+        android:layout_height="307dp"
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="52dp"
+        android:layout_marginEnd="16dp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="1.0"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/ivProfilePhoto">
+
+        <android.support.constraint.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+
+            <TextView
+                android:id="@+id/tvPersonalInformation"
+                android:layout_width="317dp"
+                android:layout_height="330dp"
+                android:layout_marginStart="8dp"
+                android:layout_marginTop="8dp"
+                android:layout_marginEnd="8dp"
+                android:text="@string/tvPersonalInformationDescription"
+                android:textSize="18sp"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
+        </android.support.constraint.ConstraintLayout>
+
+    </ScrollView>
+
+    <TextView
+        android:id="@+id/tvCommentCounter"
+        android:layout_width="44dp"
+        android:layout_height="43dp"
+        android:layout_marginStart="8dp"
+        android:layout_marginTop="108dp"
+        android:layout_marginEnd="16dp"
+        android:background="@android:color/holo_green_dark"
+        android:contentDescription="@string/tvCommentCounter"
+        android:gravity="left|right|fill_vertical|center|clip_vertical|clip_horizontal|start"
+
+        android:text="0"
+        android:textAlignment="center"
+        android:textColor="@android:color/white"
+        android:textSize="30sp"
+        android:textStyle="bold"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintHorizontal_bias="0.025"
+        app:layout_constraintStart_toEndOf="@+id/ivProfilePhoto"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="0" />
+
+    <ImageButton
+        android:id="@+id/ibCommentBubble"
+        android:layout_width="44dp"
+        android:layout_height="42dp"
+
+        android:layout_marginStart="4dp"
+        android:layout_marginTop="108dp"
+
+
+        android:adjustViewBounds="true"
+        android:background="@drawable/comment_bubble"
+        android:contentDescription="@string/ibCommentBubble"
+        android:scaleType="centerCrop"
+        android:scaleX="0.85"
+        android:scaleY="0.85"
+        android:src="@drawable/comment_bubble"
+        app:layout_constraintStart_toEndOf="@+id/tvCommentCounter"
+        app:layout_constraintTop_toTopOf="parent"
+        app:srcCompat="@android:color/holo_blue_dark"
+        tools:srcCompat="@drawable/comment_bubble" />
+
+    <TextView
```

Comment:
HINT: It would be good to dock it to constraint layout with 16dp margin on right side.